mmv1/products/privateca/api.yaml

```diff
@@ -836,7 +824,7 @@ objects:
               name: 'format'
               output: true
               description: |
-                Types of public keys that are supported. At a minimum, we support RSA, for the key sizes or curves listed: https://cloud.google.com/kms/docs/algorithms#asymmetric_signing_algorithms
+                Types of public keys that are supported. The key formats that are supported are RSA, for the key sizes or curves listed: https://cloud.google.com/kms/docs/algorithms#asymmetric_signing_algorithms
```

Comment:
I think we want to say PEM rather than RSA. In beta, we used to require explicitly specifying the key algorithm type (RSA vs. ECDSA etc..), but now we require specifying the format (PEM, etc..). As you mentioned elsewhere, we don't currently support anything other than PEM, so we could either document PEM as the only key format that's currently supported (and leave this field for future-proofing), or omit this field from here altogether and just always default the format to PEM in golang (assuming that adding it back in the future is backwards-compatible).

WDYT?

Reply:
Based on staged c.f.c docs I'm leaning towards `"The format of the public key. Currently, only PEM format is supported."` here, which'll result in `"The format of the public key. Currently, only PEM format is supported. Possible values: ["KEY_TYPE_UNSPECIFIED", "PEM"]"` WDYT?

Keeping it in schema is preferable, adding fixed-value nested fields is difficult in our generator.

Reply:
your suggestion SGTM :-)